streams/readable-streams/brand-checks.js

```diff
@@ -148,4 +148,26 @@ test(() => {
 
 }, 'ReadableStreamController.prototype.error enforces a brand check');
 
+promise_test(t => {
+
+  return Promise.all([
+    methodRejects(t, ReadableStream.prototype, 'cancel', null),
+    methodRejects(t, ReadableStream.prototype, 'cancel', undefined)
+  ]);
+
+}, 'ReadableStream brand checks do not throw on null or undefined');
+
+promise_test(t => {
+
+  return Promise.all([
+    methodRejects(t, ReadableStreamDefaultReader.prototype, 'read', undefined),
+    methodRejects(t, ReadableStreamDefaultReader.prototype, 'read', null)
+  ]);
+
+}, 'ReadableStreamDefaultReader brand checks do not throw on null or undefined');
+
+// There is no way to check ReadableStreamDefaultController brand checks for
```

Comment:
Okay, so I started over and ended up changing a lot of things. The WritableStream brand checks turned out to be a spider's nest of bugs. Typing out all those repetitive lines was taking forever, so I added *ForAll() versions of the testing functions which test several objects at the same time.

It should all be fine now. PTAL.